[2024-08-26T00:19:46.990+0700] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-26T00:19:47.117+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: scrap_mobil123.scrape_mobil123 scheduled__2024-08-24T00:00:00+00:00 [queued]>
[2024-08-26T00:19:47.125+0700] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: scrap_mobil123.scrape_mobil123 scheduled__2024-08-24T00:00:00+00:00 [queued]>
[2024-08-26T00:19:47.126+0700] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-26T00:19:47.137+0700] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): scrape_mobil123> on 2024-08-24 00:00:00+00:00
[2024-08-26T00:19:47.141+0700] {standard_task_runner.py:64} INFO - Started process 16692 to run task
[2024-08-26T00:19:47.143+0700] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'scrap_mobil123', 'scrape_mobil123', 'scheduled__2024-08-24T00:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/mobil123/scrap_mobil123.py', '--cfg-path', '/tmp/tmpswzhun_x']
[2024-08-26T00:19:47.145+0700] {standard_task_runner.py:91} INFO - Job 121: Subtask scrape_mobil123
[2024-08-26T00:19:47.205+0700] {task_command.py:426} INFO - Running <TaskInstance: scrap_mobil123.scrape_mobil123 scheduled__2024-08-24T00:00:00+00:00 [running]> on host BrightHorizons.
[2024-08-26T00:19:47.305+0700] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='scrap_mobil123' AIRFLOW_CTX_TASK_ID='scrape_mobil123' AIRFLOW_CTX_EXECUTION_DATE='2024-08-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-24T00:00:00+00:00'
[2024-08-26T00:19:47.306+0700] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-26T00:19:48.262+0700] {logging_mixin.py:188} INFO - Fetching page 1 of 2 for price category: dibawah 30 Juta
[2024-08-26T00:19:49.327+0700] {logging_mixin.py:188} INFO - Sleeping for 1.62 seconds...
[2024-08-26T00:19:50.950+0700] {logging_mixin.py:188} INFO - Fetching page 2 of 2 for price category: dibawah 30 Juta
[2024-08-26T00:19:51.725+0700] {logging_mixin.py:188} INFO - Sleeping for 2.52 seconds...
[2024-08-26T00:19:54.249+0700] {logging_mixin.py:188} INFO - Data for price category dibawah 30 Juta written to /home/miracle/mobil/scrape/output/mobil123/2024-08-26/mobil123_dibawah 30 Juta_2024-08-26.csv
[2024-08-26T00:19:55.467+0700] {logging_mixin.py:188} INFO - Fetching page 1 of 3 for price category: dibawah 40 Juta
[2024-08-26T00:19:56.675+0700] {logging_mixin.py:188} INFO - Sleeping for 2.87 seconds...
[2024-08-26T00:19:59.549+0700] {logging_mixin.py:188} INFO - Fetching page 2 of 3 for price category: dibawah 40 Juta
[2024-08-26T00:20:00.852+0700] {logging_mixin.py:188} INFO - Sleeping for 2.56 seconds...
[2024-08-26T00:20:03.418+0700] {logging_mixin.py:188} INFO - Fetching page 3 of 3 for price category: dibawah 40 Juta
[2024-08-26T00:20:04.518+0700] {logging_mixin.py:188} INFO - Sleeping for 2.58 seconds...
[2024-08-26T00:20:07.100+0700] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-26T00:20:07.100+0700] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/miracle/mobil/scrape/script/mobil123/mobil123_scrape.py", line 191, in mobil123_scrape
    with open(
FileNotFoundError: [Errno 2] No such file or directory: '/home/miracle/mobil/scrape/output/mobil123/2024-08-26/mobil123_dibawah 40 Juta_2024-08-26.csv'
[2024-08-26T00:20:07.111+0700] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=scrap_mobil123, task_id=scrape_mobil123, run_id=scheduled__2024-08-24T00:00:00+00:00, execution_date=20240824T000000, start_date=20240825T171947, end_date=20240825T172007
[2024-08-26T00:20:07.124+0700] {standard_task_runner.py:110} ERROR - Failed to execute job 121 for task scrape_mobil123 ([Errno 2] No such file or directory: '/home/miracle/mobil/scrape/output/mobil123/2024-08-26/mobil123_dibawah 40 Juta_2024-08-26.csv'; 16692)
[2024-08-26T00:20:07.144+0700] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2024-08-26T00:20:07.164+0700] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-26T00:20:07.166+0700] {local_task_job_runner.py:222} INFO - ::endgroup::
